src/components/va-accordion/va-accordion.tsx

```diff
@@ -0,0 +1,43 @@
+import { Component, Element, Host, Listen, Prop, h } from '@stencil/core';
+import { getSlottedNodes } from '../../utils/utils';
+
+@Component({
+  tag: 'va-accordion',
+  styleUrl: 'va-accordion.css',
+  shadow: true,
+})
+export class VaAccordion {
+  @Element() el!: any;
+
+  @Listen('accordionItemToggled')
+  itemToggledHandler(event: CustomEvent) {
+    const clickedItem = event.detail.target.parentNode.host;
+    // Close the other items if this accordion isn't multi-selectable
+    if (!this.multi) {
+      getSlottedNodes(this.el, 'VA-ACCORDION-ITEM')
```

Comment:
Is it standard practice to yell the slot name? :sweat_smile:

Reply:
:laughing: This is why I did that:

<details>
    <summary>The diff to get the log</summary>
    
    ```diff
    diff --git a/src/utils/utils.ts b/src/utils/utils.ts
index 9a794c1..8826330 100644
--- a/src/utils/utils.ts
+++ b/src/utils/utils.ts
@@ -3,6 +3,7 @@ export function format(first: string, middle: string, last: string): string {
 }
 
 export function getSlottedNodes(root: HTMLElement, nodeName: string): Array<Node> {
+  console.log(root.shadowRoot.querySelectorAll('slot')[0].assignedNodes());
   return root.shadowRoot
     .querySelectorAll('slot')[0]
     .assignedNodes()
     ```
     
 </details>
 
![image](https://user-images.githubusercontent.com/2008881/107996456-4eee9180-6f95-11eb-8f77-b3624501f8b7.png)


 I'll tweak the util function though so that it compares things that have been lowercased.